models/mpas_atm/model_mod.f90

```diff
@@ -7380,7 +7381,7 @@ function theta_to_tk (ens_size, theta, rho, qv, istatus)
 
 integer,                       intent(in)  :: ens_size
 real(r8), dimension(ens_size), intent(in)  :: theta    ! potential temperature [K]
-real(r8), dimension(ens_size), intent(in)  :: rho      ! dry density
+real(r8), dimension(ens_size), intent(in)  :: rho      ! dry air density [kg/m3]
```

Comment:
There are a bunch of subroutine arguments labelled as `dry density`. Should these all be `dry air density`?

Reply:
edit: it is one  in `compute_full_pressure`

Reply:
You are right. Here, rho is all 'dry air density'. Nice catch!